docs/content/cookbook/handling-custom-occ-endpoint.md

```diff
@@ -0,0 +1,182 @@
+# Handling custom OCC endpoints
+
+It is a common task to add support for a custom SAP OCC API endpoint. This guide will show you how can do it using Alokai.
```

Comment:
```suggestion
It is a common task to add support for additional SAP OCC API endpoints not covered by the Alokai integration. This guide will show you how can do it using Alokai.
```

Does this make sense? If not, ignore. But I just think clarifying what "custom" means in context of an Alokai application could be useful

Reply:
Custom means non-standard endpoint developed as customization on the back end side.